projects/npmjs.com/package.yml

```diff
@@ -10,7 +10,10 @@ provides:
   - bin/npx
 
 dependencies:
-  nodejs.org: '>=14'
+  # 20.3.x causes a error on linux+aarch64 (technically docker)
+  # sh: 1: tldr: Text file busy
+  # https://github.com/nodejs/node/issues/48444
+  nodejs.org: '>=14<20.3'
```

Comment:
we kinda need an override at a different level for this sort of thing.

an env which is “never use this version of node”, because doing this is wrong to the greater ecosystem. Not to mention we may forget to undo it.

Reply:
This is true.

Reply:
Possibly it's smarter just to override for testing, since we _know_ we're in docker there?